lib/Widget/Render/WidgetHtmlRenderer.php

```diff
@@ -468,12 +469,19 @@ private function render(
                 $widget->isValid = 0;
             }
 
+            $properties = $module->getPropertyValues();
+            $translator = null;
+
+            if (array_key_exists('lang', $properties) && !empty($properties['lang'])) {
```

Comment:
This relies on the property id being `lang` - could we be cleverer and look through the property object too?

This also means the language property needs to be on the module and not the module template - which might be right, we just need to make sure (perhaps we could add a comment to say what we're doing?)